package.json

```diff
@@ -45,7 +45,8 @@
     "testState": "node ./tests/tester -s",
     "testBlockchain": "node --stack-size=1500 ./tests/tester -b",
     "lint": "standard",
-    "test": "node ./tests/tester -a"
+    "test": "node ./tests/tester -a",
+    "build": "browserify index.js -o bundle.js"
```

Comment:
I don't think this belongs in here, since the VM is not necessarily for standalone usage